src/settings/Settings.tsx

```diff
@@ -758,6 +749,12 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         supportedLevels: LEVELS_ACCOUNT_SETTINGS,
         default: null,
     },
+    "Spaces.all_rooms_in_home": {
```

Comment:
the naming scheme for dotted settings is `Spaces.allRoomsInHome` - there used to be a style note somewhere in this file, but that seems to have gone missing.